po/oc.po

```diff
@@ -0,0 +1,578 @@
+# Translation in Occitan.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the flatseal package.
+# Quentin PAGÈS, 2022
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: flatseal\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-05-17 19:49-0400\n"
+"PO-Revision-Date: 2022-06-04 11:31+0200\n"
+"Last-Translator: Quentin PAGÈS\n"
+"Language-Team: \n"
+"Language: oc\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Generator: Poedit 3.0.1\n"
+
+#. TRANSLATORS: Don't translate this text
+#: data/com.github.tchx84.Flatseal.desktop.in:4
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:4
+msgid "Flatseal"
+msgstr "Flatseal"
```

Comment:
> TRANSLATORS: Don't translate this text

Reply:
Depends what you call untranslated...it's the same string so it's untranslated, or we should fill in the corresponding msgstr here, thus I am wondering where is there a translation key.

Reply:
Good question, looks like this is done a bit inconsistent when I look at the other translations.

Reply:
ufff, I was asleep when I wrote xD Where=>why